server/autotest_server/testers/ai/ai_tester.py

```diff
@@ -0,0 +1,116 @@
+import json
+import os
+import sys
+
+from ..tester import Test, Tester
+from ..specs import TestSpecs
+import subprocess
+from typing import Type
+from dotenv import load_dotenv
+
+
+class AiTest(Test):
+    def __init__(
+        self,
+        tester: "AiTester",
+        result: dict = None,
+    ):
+        self._test_name = result["title"]
+        self.message = result.get("message", "")
+        self.status = result["status"]
+        super().__init__(tester)
+
+    @property
+    def test_name(self) -> str:
+        """The name of this test"""
+        return self._test_name
+
+    @Test.run_decorator
+    def run(self) -> str:
+        """
+        Return a json string containing all test result information.
+        """
+        if self.status in ("success", "pass"):
```

Comment:
There's no need for two different options here; only one is used for this class.